### DISCUSSION TITLE
Is there any way to change a symbol to Unicode (Nerd Font) one in git config setting?

### DISCUSSION THREAD

**Discussion by Destroy666x** [dandavison/delta#1535] [Q&A] [answered]
Let's say I'd like to change `git config --global delta.icons.right-arrow`. Is there anything that can be done to use a Unicode character like `` (`\uEAE0`) there?

I tried:
- pasting the character directly: `""`
- `"\uEAE0"`
- `"\U0000EAE0"`

But the output of the 1st looks like this:
![image](https://github.com/dandavison/delta/assets/3606072/20e3d812-7c7c-4b9d-805a-2cba92cc0725)
while the rest is just text.

**Comment by injust** [comment, marked as answer]
There's some discussion in https://github.com/dandavison/delta/issues/1616 about this.
